controller/OrderController.go

```diff
@@ -0,0 +1,35 @@
+package controller
+
+import (
+	"fmt"
+	"github.com/gin-gonic/gin"
+	"log"
+	"net/http"
+	"stlab.itechart-group.com/go/food_delivery/courier_service/dao"
+)
+
+
+// @Summary UpdateOrder
+// @Tags order
+// @Description assign order to courier
+// @ID UpdateOrder
+// @Accept  json
+// @Produce json
+// @Param input body dao.Order true "id courier/id order"
+// @Success 204
+// @Failure 400,404 {string} string
```

Comment:
having status 404 for update is a good idea, but I don't see that you really return such status from the handler